samples/interop/src/main/kotlin/com/slack/circuit/sample/interop/SimpleCounterPresenter.kt

```diff
@@ -41,25 +41,17 @@ class SimpleCounterPresenter {
 }
 
 fun SimpleCounterPresenter.asCircuitPresenter(): Presenter<CounterScreen.State> {
+  // TODO why is this key necessary? The returned presenter is a new instance
```

Comment:
Is it? I removed the `key` and ran it through the debugger, its creating a new listener each time 🤔

Reply:
I was seeing it reuse for some reason, was driving me nuts. I'll try again

Reply:
Hmm, `CircuitInteropTest` still fails if I don't use a key 🤔

Reply:
```suggestion
  // Necessary due to the positional key being the same
```